### PR TITLE
fix: Replace grant for the whole Lists controller with one just for getListElements

### PR DESCRIPTION
**Pull request by hectoras**
**Associated Jira issue:** [AUT-1927](https://oat-sa.atlassian.net/browse/AUT-1927)

This PR replaces the grant for `PropertyManagers` for accessing the whole BackOffice List controller and grants access just to the `getListElements` endpoint, which is enough for Manage Schema pages.

- [x] Verify the migration fixes the issue for a existing environment
- [x] Verify the fix in the manifest works on a fresh installation

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -40,7 +41,7 @@
         [
             AccessRule::GRANT,
             'http://www.tao.lu/Ontologies/TAO.rdf#PropertyManagerRole',
-            ['controller' => 'oat\taoBackOffice\controller\Lists'],
+            ['act' => Lists::class .'@getListElements'],
```
Comment (gabrielfs7) [inline manifest.php:44]: Lets also check if the previous bug fixed for UDIR wont come back after this change. This change will be deployed to them, so we might run this migrations in the dev environment for UDIR and test it.

Comment (hectoras) [reply manifest.php:44]: I've tested this in dev & testing envs for UDIR and the call to getListElements is working fine both for a admin and author users.

**Review by gabrielfs7 — APPROVED** [review]
Approved since it works on local env and did not crash UDIR env

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: manifest.php [hunk 2]
```diff
@@ -40,7 +41,7 @@
         [
             AccessRule::GRANT,
             'http://www.tao.lu/Ontologies/TAO.rdf#PropertyManagerRole',
```
Comment (wazelin) [inline manifest.php:43]: Just a suggestion, but I guess you can also replace this value with `TaoRoles` constant.
```suggestion
            TaoRoles::PROPERTY_MANAGER,
```

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

File: manifest.php (hunk 2 above)
Comment (hectoras) [reply manifest.php]: You are right, changed in 808ef6115e4d46b22245c9cfe7e88abd42ec6c45. Thanks!

**Comment by AntonNeviarouski** [comment]
QA verified

**Comment by hectoras** [comment]
Released as [v6.10.0](https://github.com/oat-sa/extension-tao-backoffice/releases/tag/v6.10.0).

**Comment by hectoras** [comment]
Backported as [v6.6.2](https://github.com/oat-sa/extension-tao-backoffice/releases/tag/v6.6.2).

**Comment by hectoras** [comment]
Backported as [v6.9.0.1](https://github.com/oat-sa/extension-tao-backoffice/releases/tag/v6.9.0.1).

**Pull request merged**